### PR TITLE
Issue 13527 - ddoc website documentation does not match the current built-in symbols

### PR DESCRIPTION
**Pull request by nomad-software**
All current built-in ddoc symbols have been added to the documentation. I've also removed the v2.067.0 specific section and rolled it into the main list. It doesn't really make sense to have this as all the other symbols are not specified by versions. I've also aligned the text within the list to be more readable.

Preview of changes made: http://nomad.so/misc/d/dlang.org/issue_13527/ddoc.html

Bugzilla: https://issues.dlang.org/show_bug.cgi?id=13527

### CODE REVIEW

File: ddoc.dd [hunk 1]
```diff
@@ -684,43 +684,48 @@ $(P
...
+LPAREN     = $(LPAREN)
+RPAREN     = $(RPAREN)
+BACKTICK   = $(BACKTICK)
+DOLLAR     = $(DOLLAR)
+DEPRECATED = $(DOLLAR)0
+LT         = $(LT)
+GT         = $(GT)
+AMP        = $(AMP)
+
+RED    = $(LT)font color=red$(GT)$(DOLLAR)0$(LT)/font$(GT)
```
Comment (MartinNowak) [inline ddoc.dd]: Never heard of the `font` element before, it's discouraged http://www.w3.org/wiki/HTML/Elements/font, well...

Comment (CyberShadow) [inline ddoc.dd]: Heh, `<font>` predates CSS :)

File: ddoc.dd [hunk 2]
```diff
@@ -684,43 +684,48 @@ $(P
...
+LINK       = $(LT)a href="$(DOLLAR)0"$(GT)$(DOLLAR)0$(LT)/a$(GT)
+LINK2      = $(LT)a href="$(DOLLAR)1"$(GT)$(DOLLAR)+$(LT)/a$(GT)
+LPAREN     = $(LPAREN)
+RPAREN     = $(RPAREN)
+BACKTICK   = $(BACKTICK)
+DOLLAR     = $(DOLLAR)
+DEPRECATED = $(DOLLAR)0
+LT         = $(LT)
+GT         = $(GT)
+AMP        = $(AMP)
```
Comment (MartinNowak) [inline ddoc.dd]: Those 3 are not part of the compiler predefines.

File: ddoc.dd [hunk 3]
```diff
@@ -731,70 +736,50 @@ DDOC = $(LT)html$(GT)$(LT)head$(GT)
...
+DDOC_PARAM_ID         = $(DOLLAR)(TD $(DOLLAR)0)
+DDOC_PARAM_DESC       = $(DOLLAR)(TD $(DOLLAR)0)
+DDOC_ANCHOR           = $(LT)a name="$(DOLLAR)1"$(GT)$(LT)/a$(GT)
+DDOC_PSYMBOL          = $(DOLLAR)(U $(DOLLAR)0)
+DDOC_PSUPER_SYMBOL    = $(DOLLAR)(U $(DOLLAR)0)
+DDOC_KEYWORD          = $(DOLLAR)(B $(DOLLAR)0)
+DDOC_PARAM            = $(DOLLAR)(I $(DOLLAR)0)
+DDOC_BACKQUOTED       = $(DOLLAR)(D_INLINECODE $(DOLLAR)0)
+DDOC_BLANKLINE        = $(DOLLAR)(BR)$(DOLLAR)(BR)
+DDOC_COMMENT          = $(LT)!-- $(DOLLAR)0 --$(GT)
```
Comment (MartinNowak) [inline ddoc.dd]: Would be nice to keep them in the same order as in dmd.
https://github.com/D-Programming-Language/dmd/blob/312a931d03d119fa6e6af4099b5fb1beb0d71ebc/src/doc.c#L193

File: ddoc.dd (hunk 1 above)
Comment (nomad-software) [inline ddoc.dd]: This should be raised as another ticket as the goal of this one is to add missing symbols.

**Comment by nomad-software** [comment]
Removed the 3 symbols not defined in the compiler and re-ordered the list to match dmd.

**Comment by nomad-software** [comment]
Issue raise for the outdated tags: https://issues.dlang.org/show_bug.cgi?id=14473

**Pull request merged**